### PR TITLE
Official transformation of Hessen for the land survey register (ALKIS).

### PR DESCRIPTION
**Pull request by ChristianMaierHVBG**
This is the official transformation of Hessen for the land survey register (ALKIS).

I followed all the steps of contributing to PROJ-data. Hopefully everything works out. 

I also registered the transformation before at epsg.org. You can find it here: https://epsg.org/transformation_9940/DHDN-to-ETRS89-11.html. I am not sure if this is enough for sql-files. Maybe there is still a manual addition necessary

### CODE REVIEW

File: de_hvbg/de_hvbg_README.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+The files in this section result from the conversion of datasets originating from [Hessische Verwaltung für Bodenmanagement und Geoinformation](https://www.hvbg.hessen.de)
+
+## Included grids
+
+### Germany (Hessen): DE_DHDN -> ETRS89
+
+*Source*: [HVBG Downloadcenter](https://www.gds.hessen.de/INTERSHOP/web/WFS/HLBG-Geodaten-Site/de_DE/-/EUR/ViewData-Start/1394640044?JumpTarget=ViewRequisitionProcess-Restart&IsOnlineBasket=false&ProductUUID=B9MKFwNVKxMAAAF.AeJVXZuMhttps://www.gds.hessen.de/INTERSHOP/web/WFS/HLBG-Geodaten-Site/de_DE/-/EUR/ViewData-Start/1394640044?JumpTarget=ViewRequisitionProcess-Restart&IsOnlineBasket=false&ProductUUID=B9MKFwNVKxMAAAF.AeJVXZuM)
+
+*Format*: GeoTIFF converted from NTv2                                                                                                                    
+*License*: Free redistribution is allowed and welcome
```
Comment (rouault) [inline de_hvbg/de_hvbg_README.txt:12]: This is too vague as a license. We require the use of a well-known open data license such as the ones in https://github.com/OSGeo/PROJ-data#about-the-data-package. Other German geodetic agencies also sometimes use the "Data licence Germany - attribution - version 2.0"

The copyright_and_licenses.csv files should also be modified

**Review by rouault — COMMENTED** [review]
-

File: de_hvbg/de_hvbg_README.txt (hunk 1 above)
Comment (rouault) [reply de_hvbg/de_hvbg_README.txt:12]: > we don't have any licence defined for it and there is nobody who is responsible for this topic

Are you connected with the team that produced the file? Who is the owner of the intellectual property with it ? If it is you (your organization), then you can freely decide of the license for the data you deliver. CC-BY-4.0 (https://creativecommons.org/licenses/by/4.0/) is a quite popular open data license that a lot of geodetic agencies use for their grids. It cover all the open data freedoms while crediting the copyright owner of the data.

**Comment by rouault** [comment]
@ChristianMaierHVBG It seems you might be strugling with formatting of the readme ? For some reason, lines starting with `*foo*`` should be terminated with 2 spaces. Anyway don't worry too much about that. We can fix that.
The main thing to do at that point would be to add  new lines in copyright_and_licenses.csv for the readme and the grid

**Comment by ChristianMaierHVBG** [comment]
Alright, we (my organization) decided to use the "Data licence Germany - attribution - version 2.0" aswell. I also modified the copyright_and_licenses.csv. Is there anything else I have to do now? Like you already mentioned, I am not really into GitHub, this is the first time I use it.

**Comment by rouault** [comment]
> I also modified the copyright_and_licenses.csv

you probably forgot to git commit / push those changes

> Is there anything else I have to do now?

that should be fine otherwise for this repository. In The OSGeo/PROJ repository, you also need to add a new line to link the .gsb grid name registerd in EPSG to the PROJ GeoTIFF one. Cf https://github.com/OSGeo/PROJ/blob/master/data/sql/grid_alternatives.sql#L99

**Comment by ChristianMaierHVBG** [comment]
Now it should be updated.

File: copyright_and_licenses.csv [hunk 2]
```diff
@@ -71,6 +71,8 @@ de_adv_BETA2007.tif,ADV,Free redistribution is allowed and welcome,,
 de_adv_README.txt,Disclaimed,Public domain,,
 de_geosn_NTv2_SN.tif,Staatsbetrieb Geobasisinformation und Vermessung Sachsen (GEOSN),Data licence Germany - attribution - version 2.0,,
 de_geosn_README.txt,Disclaimed,Public domain,,
+de_hvbg_hessen_HeTA2010.tif,Hessisches Landesamt für Geoinformation und Bodenmanagement (HVBG),Data licence Germany - attribution - version 2.0,,
```
Comment (rouault) [inline copyright_and_licenses.csv:74]:
```suggestion
de_hvbg_hessen_HeTA2010.tif,Hessisches Landesamt für Geoinformation und Bodenmanagement (HVBG),Data licence Germany - attribution - version 2.0,1.14,
```

**Pull request merged**